src/main/java/com/bugsnag/Configuration.java

```diff
@@ -39,6 +50,8 @@
         if (ServletCallback.isAvailable()) {
             addCallback(new ServletCallback());
         }
+        sessionDelivery = new AsyncHttpDelivery();
+        ((HttpDelivery) sessionDelivery).setEndpoint("https://sessions.bugsnag.com");
```

Comment:
We should probably have the default `setEndpoint` configured for sessions and notify in the same place. Currently we have `https://notify.bugsnag.com` set as the default error reporting endpoint in `SyncHttpDelivery.java` but session reporting defaults are configured here.

Also, should we have this URL in a a `static final` for easier visibility of default values all in one place?